modules/virt-fips-support.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
```

Comment:
You will need to remove this file from the repo by using the `git rm virt-fips-support.adoc` command.